pkg/common/error/error.go

```diff
@@ -45,81 +45,86 @@ const (
 	/**
 	 * Io transaction exception code.
 	 */
-
+	IO = 3
 	/**
 	 * Branch rollback failed retriable transaction exception code.
 	 */
-	TransactionExceptionCodeBranchRollbackFailedRetriable = 3
```

Comment:
change `TransactionExceptionCodeBranchRollbackFailedRetriable = 3` to `TransactionExceptionCodeBranchRollbackFailedRetriable = TransactionExceptionCode(3)`, the same as below